docs/docs/docs/getting-started/installation.md

```
@@ -307,37 +307,42 @@
+#### CLI Startup (Development)
 
-Use the command `docker compose` command to cleanly shutdown the dev container
+After a successful installation, use these commands to start the dev container.
 
-```
-docker compose down
-```
+1. To run in attached Mode
 
-#### CLI Startup (Development)
+   ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)